src/twisted/python/htmlizer.py

```diff
@@ -21,7 +21,7 @@ class TokenPrinter:
 
     currentCol, currentLine = 0, 1
     lastIdentifier = parameters = 0
-    encoding = "utf-8"
+    encoding = b"utf-8"
```

Comment:
You can't use a byte string with encode on Python 3, right?  So this is only valid in Python 2 where `b"utf-8"` is the same thing as `"utf-8"`.

Furthermore, [it looks like `tokenize` on Python 3 always emits an `ENCODING` token](https://docs.python.org/3/library/tokenize.html#tokenize.ENCODING):

> The first token returned by tokenize() will always be an ENCODING token.

Overall It looks like the effect of this `bytes` literal and the `hasattr` check below would be more clearly express with `if _PY3`.